src/mmw/sass/components/_modals.scss

```diff
@@ -31,7 +31,7 @@
 .modal-basic {
 
   .modal-content {
-    padding: 1rem;
+    padding: 3rem;
```

Comment:
Hmmm. We should say "Make Public" everywhere. If it is an easy fix, I would recommend including it here.